app/controllers/api/v1/members_controller.rb

```diff
@@ -0,0 +1,74 @@
+class Api::V1::MembersController < ApplicationController
+  include AuthenticationCheck
+
+  before_action :is_user_logged_in
+  before_action :set_member, only: [:show, :update, :destroy]
+
+  # GET /members
+  def index
+    @members = Member.where(user_id: current_user.id)
+    render json: {members: @members}
+  end
+
+  # GET /members/:id
+  def show
+    if check_access
+      render json: @member, status: 201
```

Comment:
201 is the status code for when something is created (<https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201>) but this method simply displays existing data. You could update the number:

```suggestion
      render json: @member, status: 200
```

or use the appropriate response symbol that Rails recognizes:

```suggestion
      render json: @member, status: :ok
```

or simply omit the `status` entirely because Rails defaults to `:ok`/200.

```suggestion
      render json: @member
```